scripts/unmount.sh

```diff
@@ -11,5 +11,19 @@ if [ "$(readlink /proc/self/ns/mnt)" != "$(readlink /proc/1/ns/mnt)" ]; then
   exit 1
 fi
 
-umount -v /mnt/runtime/write/emulated/0/the_binding
-umount -v /mnt/my_drive
+android_version=$(getprop ro.build.version.release)
+
+if [ $android_version -gt 10 ]; then
+  # for Android 11+
+  internal_binding_dir="/mnt/pass_through/0/emulated/0/the_binding"
```

Comment:
i believe this causes an issue where `unmount.sh` is no longer compatible with `mount_ext4.sh`

we might have to split `unmount.sh` into two separate scripts or maybe have a separate set of scripts for specific android versions.